README.md

```diff
@@ -1,33 +1,29 @@
 #  Sustainable STEAM-driven Development of Regions
 
-We are uncovering better ways of developing regions through STEM by doing it and helping others do it.
+We are uncovering better ways of developing regions through STEM by doing it and helping others do it. We promote the teaching of STEAM as key drivers of sustainable development and social well-being of a region.
 
 Through this work we have come to value:
 
 <p style="text-align: center;">
-Alliances and collaborations over knowledge silos<br>
-Attracting sustainable STEM talent over wasting it outside the region<br>
-Embrace innovation over conservative strategies<br>
-Promote sustainable STEM development over speculative bubbles<br>
-People and relationships over goals and objectives<br>
-Think beyond the limits established over initiatives and conformist thoughts<br>
+- Alliances and collaborations over knowledge silos<br>
+- Attracting sustainable STEM talent over wasting it outside the region<br>
+- Embrace innovation over conservative strategies<br>
+- Promote sustainable STEM development over speculative bubbles<br>
+- People and relationships over goals and objectives<br>
+- Think beyond the limits established over initiatives and conformist thoughts<br>
 </p>
 
 That is, while there is value in the items on the right, we value the items on the left more.
 
 We, as implementers, follow these principles:
 
-1. The highest priority is to make the represented region visible on the international STEM landscapes.
-1. All initiatives are driven by transparency, honesty, and respect people.
```

Comment:
- [ ] I like this one, I see it really important in the current era, with so much noise, and people fighting each other. I would propose keeping it.